### PR TITLE
[fv_all] Add fv_all package and custom build

### PR DESCRIPTION
**Pull request by mcdurdin**
This pull request creates a custom package which includes all the FirstVoices Windows keyboards, in order to prepare package for Windows FirstVoices Keyboards deployment. The package is constructed dynamically from the fv/ folder so includes all keyboards that are available at time of build.

The package is not really intended for use with Keyman Desktop; the standalone keyboard packages are better suited to that. Instead it is mostly for use by the FirstVoices Keyboards app.

### CODE REVIEW

File: resources/compile.sh [hunk 1]
```diff
@@ -170,6 +170,10 @@ function build_keyboard {
   if [ -f build.sh ]; then
     # We call this build.sh and assume it generates the
     # right stuff
+    if [[ ! -z "$PROJECT_TARGET_TYPE" ]]; then
```
Comment (mcdurdin) [inline resources/compile.sh]: This adds support for `-T` to the custom build script call.

**Comment by mcdurdin** [comment]
Realised this morning that I still need to add the fonts to fv_all.kps.in -- so not quite ready to merge...

**Comment by mcdurdin** [comment]
This is ready for re-review.

**Comment by LornaSIL** [comment]
I reviewed the changes to sencoten and to dakelh and those are fine. I don't have Keyman 12 installed. Do I need to switch to Keyman 12 to test this package? Or trust you?

**Comment by mcdurdin** [comment]
You should be able to test the package in earlier versions of Keyman; it is
not really intended for installation into Keyman Desktop, however, but as
part of the FirstVoices Keyboards distribution.

**Comment by LornaSIL** [comment]
* It's looking for a .kps file that doesn't exist. There is a ```fv_all.kps.in```, but it has a few things I'm not used to: ```@FILES``` and ```@KEYBOARDS```

* I don't know what the ```parse_kps.pl``` file is supposed to do.

I'm just not sure how to build it. I can't build it in Keyman Developer. When I say "Build Packages" I get this message:
```
fv_all.kps: Compiling package fv_all.kps...
fv_all.kps: Error: You need to fill in the package name before building.
fv_all.kps: Error: 'C:\0WSTech\working\keyboards\release\packages\fv_all\source\fv_all.kps' was not compiled successfully.
```
I see the ```build.sh``` but I'm not setup to run that. So, how should I test, or is something in the files wrong?

**Comment by mcdurdin** [comment]
This package can only be built with the build.sh script :(  it is
dynamically constructed from all the keyboards in the fv folder.

From git bash (it should be installed already), if you had your keyboard
repo in C:\path\to\keyboards\repo you’d run the top level build.sh as
follows:

    cd /c/path/to/keyboards/repo
    ./build.sh release/packages/fv_all

**Comment by LornaSIL** [comment]
I don't know what I'm doing wrong, but I typed ``` ./build.sh release/packages/fv_all``` and it does build the .kmp, but the only thing in the package is the fonts, readme.htm, kmp.inf and kmp.json.

**Comment by mcdurdin** [comment]
For the purposes of documentation, prior to `./build.sh release/packages/fv_all` you need to build the standalone fv_* packages, using, for example, `./build.sh release -start fv`. The CI process does this already.

**Pull request merged**